README.md

```diff
@@ -1 +1,218 @@
+},
+```
+
+### Require the sample data modules
+
+Add the sample data modules to the `require` section of the `composer.json` file to install them into Magento.
+
+```json
+"magento/module-catalog-sample-data-venia": "dev-master",
+"magento/module-configurable-sample-data-venia": "dev-master",
+"magento/module-customer-sample-data-venia": "dev-master",
+"magento/module-sales-sample-data-venia": "dev-master",
+"magento/module-tax-sample-data-venia": "dev-master",
+"magento/sample-data-media-venia": "dev-master",
+```
+
+## Modify `.gitignore` file
+
+Add the following frontend dependency entries to the `.gitignore` file to track them in your repository.
+
+```text
+!package.json
+!yarn.lock
+```
+
+## Install dependencies
+
+### Frontend dependencies
+
+Install the frontend dependencies specified in the `package.json` file.
+
+```sh
+yarn install
+```
+
+A successful installation creates a `yarn.lock` file in your project directory.
+
+### Composer dependencies
+
+Install the new Composer dependencies added to the `composer.json` file.
+
+```sh
+composer update
+```
+
+## Specify Cloud server environment
+
+### Install Yarn
+
+If your project uses Yarn, which is the case for `venia-concept`, add the following entry to the `.magento.app.yaml` file.
+
+```yaml
+dependencies:
+  nodejs:
+    yarn: "*"
+```
+
+### Update build hooks
+
+The `venia-concept` storefront uses a newer version of Node than the default environment provides.
+It also uses Yarn to install its dependencies.
+
+The following update to the `hooks.build` section of `.magento.app.yaml` installs NVM, installs the latest LTS release of Node, and uses Yarn to install the project dependencies.
+
+```yaml
+hooks:
+  # We run build hooks before your application has been packaged.
+  build: |
+    set -e
+
+    unset NPM_CONFIG_PREFIX
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | dash
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+    nvm install --lts=dubnium
+
+    yarn install
+
+    php ./vendor/bin/ece-tools build:generate
+    php ./vendor/bin/ece-tools build:transfer
+```
+
+### Add required environment variables
+
+Use the Magento Cloud GUI or modify the `variables.env` entry in `.magento.app.yaml` to add any required environment variables.
+
+The following table lists the required environment variables for the Venia storefront:
+
+| Name                                                    | Value                                                       |
+| ------------------------------------------------------- | ----------------------------------------------------------- |
+| `CONFIG__DEFAULT__WEB__UPWARD__PATH` | `/app/node_modules/@magento/venia-concept/venia-upward.yml` |
+| `NODE_ENV`                                              | `production`                                                |
+| `MAGENTO_BACKEND_URL` | `https://[your-cloud-url-here]` |
```

Comment:
Need to add `USE_FASTLY`. Should be `0` in integration branches, `1` if it's a production or staging branch with Fastly configured
Should we also mention `BRAINTREE_TOKEN`?

Reply:
Yeah, we need both those things--plus, we should clarify that `CONFIG__DEFAULT__WEB__UPWARD__PATH` must be the absolute path of the UPWARD file, and that the provided one happens to be Venia's. (As opposed to there being a standard name for the UPWARD file, which there intentionally isn't.)

Reply:
Added.

@jcalcaben can you review formatting, and suggest an edit if I butchered anything.

Reply:
The formatting looks good to me when I view the rendered file in GitHub. 👍